### ISSUE TITLE
Julia v1.0 support?

### ISSUE THREAD

**Issue by dlfivefifty** [JuliaIntervals/ValidatedNumerics.jl#286]
I tried using this but get the following errors:
```julia
julia> using ValidatedNumerics
[ Info: Precompiling ValidatedNumerics [d621b6e3-7715-5857-9c6f-c67000ef6083]
ERROR: LoadError: LoadError: LoadError: UndefVarError: @doc_str not defined
Stacktrace:
 [1] top-level scope
 [2] include at ./boot.jl:317 [inlined]
 [3] include_relative(::Module, ::String) at ./loading.jl:1038
 [4] include at ./sysimg.jl:29 [inlined]
 [5] include(::String) at /Users/sheehanolver/.julia/packages/IntervalConstraintProgramming/08Yfb/src/IntervalConstraintProgramming.jl:3
 [6] top-level scope at none:0
 [7] include at ./boot.jl:317 [inlined]
 [8] include_relative(::Module, ::String) at ./loading.jl:1038
 [9] include(::Module, ::String) at ./sysimg.jl:29
 [10] top-level scope at none:2
 [11] eval at ./boot.jl:319 [inlined]
 [12] eval(::Expr) at ./client.jl:389
 [13] top-level scope at ./none:3
in expression starting at /Users/sheehanolver/.julia/packages/IntervalConstraintProgramming/08Yfb/src/ast.jl:4
in expression starting at /Users/sheehanolver/.julia/packages/IntervalConstraintProgramming/08Yfb/src/ast.jl:4
in expression starting at /Users/sheehanolver/.julia/packages/IntervalConstraintProgramming/08Yfb/src/IntervalConstraintProgramming.jl:27
ERROR: LoadError: Failed to precompile IntervalConstraintProgramming [138f1668-1576-5ad7-91b9-7425abbf3153] to /Users/sheehanolver/.julia/compiled/v1.0/IntervalConstraintProgramming/DCiIm.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] macro expansion at ./logging.jl:313 [inlined]
 [3] compilecache(::Base.PkgId, ::String) at ./loading.jl:1184
 [4] _require(::Base.PkgId) at ./logging.jl:311
 [5] require(::Base.PkgId) at ./loading.jl:852
 [6] macro expansion at ./logging.jl:311 [inlined]
 [7] require(::Module, ::Symbol) at ./loading.jl:834
 [8] include at ./boot.jl:317 [inlined]
 [9] include_relative(::Module, ::String) at ./loading.jl:1038
 [10] include(::Module, ::String) at ./sysimg.jl:29
 [11] top-level scope at none:2
 [12] eval at ./boot.jl:319 [inlined]
 [13] eval(::Expr) at ./client.jl:389
 [14] top-level scope at ./none:3
in expression starting at /Users/sheehanolver/.julia/packages/ValidatedNumerics/VqajE/src/ValidatedNumerics.jl:11
ERROR: Failed to precompile ValidatedNumerics [d621b6e3-7715-5857-9c6f-c67000ef6083] to /Users/sheehanolver/.julia/compiled/v1.0/ValidatedNumerics/EHj5c.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] macro expansion at ./logging.jl:313 [inlined]
 [3] compilecache(::Base.PkgId, ::String) at ./loading.jl:1184
 [4] _require(::Base.PkgId) at ./logging.jl:311
 [5] require(::Base.PkgId) at ./loading.jl:852
 [6] macro expansion at ./logging.jl:311 [inlined]
 [7] require(::Module, ::Symbol) at ./loading.jl:834

```

**Comment by dpsanders** [comment]
ValidatedNumerics.jl is just a metapackage that reexports the other packages. Since they are not all completely ready yet, neither is this one.

So far, IntervalArithmetic.jl and IntervalRootFinding.jl have 1.0 compatible releases. Could you please try using those directly? The other packages have a branch dps/v0.7 that should work.

**Comment by dpsanders** [comment]
Fixed by #287.

**Comment by dpsanders** [comment]
Fixed in #289.
